Sources/PointFree/Bootstrap.swift

```diff
@@ -54,6 +54,9 @@ private let loadEnvVars =
       #else
       let allEpisodes = allPublicEpisodes + allPrivateEpisodes
       #endif
+
+      assert(allEpisodes.count == Set(allEpisodes.map(^\.id)).count)
+      assert(allEpisodes.count == Set(allEpisodes.map(^\.sequence)).count)
```

Comment:
Added a good ol' runtime error! This should help us avoid the email glitch from last time.